misc/testerina/modules/testerina-core/src/main/java/org/ballerinalang/testerina/core/BTestRunner.java

```diff
@@ -452,26 +451,26 @@ private static void resolveFunctions(TestSuite suite) {
                 test.setDataProviderFunction(functions.stream().filter(e -> e.getName().equals(test.getDataProvider()
                 )).findFirst().map(func -> {
                     // TODO these validations are not working properly with the latest refactoring
-                    if (func.getbFunction().getReturnTypeNode() != null) {
```

Comment:
Why we comment out ordering tests

Reply:
This resolveFunction method gets bFunction from TesterinaFunction class. The functionality of the if clause already commented and only else condition was there. It is also to return an exception. The idea was to comment on them to remove the dependency of bFunction from TesterinaFunctions.